http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -126,6 +132,16 @@ public enum RelativeUrlsUsed {
     protected boolean verifyTokenAudience = false;
 
     protected String tokenSignatureAlgorithm = DEFAULT_TOKEN_SIGNATURE_ALGORITHM;
+    protected String authenticationRequestFormat;
+    protected String requestSignatureAlgorithm;
+    protected String requestEncryptAlgorithm;
+    protected String requestContentEncryptionMethod;
```

Comment:
We should be consistent with the use of `request` vs `requestObject`. I think using `requestObject` makes sense since that matches the actual attribute names too.